Jenkinsfile

```diff
@@ -88,7 +88,7 @@ pipeline {
 
                     params = [
                         string(name: 'CHANNEL', value: params.CHANNEL),
-                        string(name: 'BUILD_TYPE', value: params.BUILD_TYPE),
+                        string(name: 'BUILD_TYPE', value: PLATFORM == 'android' ? 'Release' : params.BUILD_TYPE),
```

Comment:
If someone starts a manual build with Static for example wouldn't that be ignored?

Reply:
Should be good enough for now. Let's have like this for a while, maybe we'll have to go back to Static at some point.